source/solvers/fluid_dynamics_matrix_free_operators.cc

```diff
@@ -1468,6 +1538,10 @@ NavierStokesStabilizedOperator<dim, number>::do_cell_integral_local(
               // +(v,(u·∇)δu + (δu·∇)u)
               value_result[i] += gradient[i][k] * previous_values[k] +
                                  previous_gradient[i][k] * value[k];
+
+              // -(v, (u_ale·∇)δu)
+              if (this->enable_mortar)
```

Comment:
might be a good idea to make a local bool that is const with the enable_mortar value in the function instead of calling the this-> every time. I think it can have an impact